lib/rouge/lexers/ssh.rb

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class SSH < RegexLexer
+      tag 'ssh'
+
+      title "SSH Config File"
+      desc 'A lexer for SSH configuration files'
+      filenames 'config', 'ssh_config'
```

Comment:
I worry that `.config` is a very generic file extension that other files might use. Do you think it would be fine just to leave this as `ssh_config`?

Reply:
It’s a file named `config` rather than `.config`, as in `~/.ssh/config`, but your point is still valid. Happy to remove if you want

Reply:
Oops—right you are. But yes, I think if you could remove it, that would be good.

Reply:
Added a commit to fix both requests

Reply:
Coming in way late to this conversation, but I'm curious (I haven't written a lexer before)... could one of the filenames be `.ssh/config` ? Would that be a way to detect the the user filename?

Thanks!

Reply:
The guesser uses the basename rather than the full path, so would only match on `config` not `.ssh/config`.

Reply:
Ah, ok thanks!

I know some other things (VS Code I think) are able to search in the path, and handle that... wasn't sure if this could.

Reply:
@IanLee1521 I'm not sure of your use case but if it's concern about GitLab correctly identifying files, you can specifically tell GitLab what language files are using `.gitattributes` (see [here](https://docs.gitlab.com/ee/user/project/git_attributes.html)).